server/index.js

```diff
@@ -50,7 +50,7 @@ export default class Server {
 
   handleRequest (req, res, parsedUrl) {
     // Parse url if parsedUrl not provided
-    if (!parsedUrl) {
+    if (!parsedUrl || typeof parsedUrl !== 'object') {
```

Comment:
I believe @rauchg once said we should handle the use case (`router.get('*', handle)`)